cli/src/commands/subgraph/commands/check.ts

```diff
@@ -59,11 +63,15 @@ export default (opts: BaseCommandOptions) => {
       }
     }
 
+    // submit an empty schema in case of a delete check
+    const schema = schemaFile ? await readFile(schemaFile) : Buffer.from('');
+
     const resp = await opts.client.platform.checkSubgraphSchema(
       {
         subgraphName: name,
-        schema: await readFile(schemaFile),
+        schema,
         gitInfo,
+        checkDeletion: options.delete,
```

Comment:
Can we call it `delete`? Check is already part of the RPC method.

Reply:
Done